docker/runners/runner-monitor.sh

```
@@ -0,0 +1,165 @@
+# transitions (healthy → unhealthy/down). Resolves silently when all recover.
+# Uses a state file to prevent alert spam.
+
+set -euo pipefail
+
+# ---------------------------------------------------------------------------
+# Config
+# ---------------------------------------------------------------------------
+EXPECTED_RUNNERS=10
+STATE_FILE="/tmp/runner-monitor-state.json"
+COMPOSE_DIR="$HOME/.omnibase/runners/docker"
+COMPOSE_FILE="${COMPOSE_DIR}/docker-compose.runners.yml"
```

Shellcheck (0.11.0)

[warning] COMPOSE_FILE appears unused. Verify use (or export if used externally).

(SC2034)